samples/sample-sync-doc-definitions.js

```diff
@@ -457,6 +458,7 @@ function() {
       typeFilter: function(doc, oldDoc) {
         return new RegExp('^paymentRequisition\\.[A-Za-z0-9_-]+$').test(doc._id);
       },
+      cannotReplace: true,
```

Comment:
All the property validators of this type make it effectively immutable; however, the document itself cannot be marked as `immutable` because authorized users should be allowed to delete payment requisitions after they're created.